ister_gui.py

```diff
@@ -1935,6 +1949,14 @@ def handler(self, config):
             return exc
         return self._action
 
+    def _erase_dup_mount(self, point, part):
+        for idx, chc in enumerate(self.choices):
+            if '{} '.format(point) in chc and '{} '.format(part) not in chc:
```

Comment:
But again that's for the preformatted handling. It seems awkward to do this post format.